### DISCUSSION TITLE
MSTR CSP issue | When v7.0.0 will be released officially

### DISCUSSION THREAD

**Discussion by suozixie233** [Turfjs/turf#2495] [Q&A] [answered]
Hi, I'm from MicroStrategy.Inc, and we are using Turf in our mapbox features.
Now we are going to achive L3 CSP in our product while the newest release v6.5.0 has ```new Function``` abuses, however, the bundle from **v7.0.0-alpha** has solved the problem.
We are expecting an official release of v7.0.0, so we can adopt it in our product with no CSP issues.
Will the v7.0.0 be released soon?
- [X] The version of Turf you are using, and any other relevant versions.
- [ ] GeoJSON data as a [gist file](https://gist.github.com/) or [geojson.io](http://geojson.io/) (filename extension must be `.geojson`).
- [ ] Snippet of source code or for complex examples use [jsfiddle](https://jsfiddle.net/).

**Comment by twelch** [comment]
Hi @suozixie233  This question has already been asked an answered.  Is there anything stopping you from using the alpha release at this time?

See https://github.com/Turfjs/turf/discussions/2491

**Reply by suozixie233** [reply]
@twelch Thanks for the quick responding.
We have two concerns about using the alpha version:
1. We have to build our own turf.min.js if we’re gonna use v7.0.0-alpha.1, is there any difference in the building process from the future official release(CDN) bundle? Will the two bundles be exactly the same?
2. Since the v7 hasn’t been released officially yet, will there be any compliance issues if we use it for commercial purposes?

**Reply by twelch** [reply, marked as answer]
I could be wrong but I feel like all published versions including alphas, include a minified build.  I'm not sure how you consume turf, if it's through a CDN, then I think you'll have to wait for the stable release.

There's no issue with you using alpha versions for commercial purposes.  That said, it's up to you to make sure it is working as expected, and providing accurate enough answers for your need.

**Reply by suozixie233** [reply]
Then I think we will adopt the v7, thanks for your help! @twelch

**Comment by smallsaucepan** [comment]
In case you weren't aware @suozixie233, v7 of Turf has now been released 👍
